packages/fluentui/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -150,7 +101,6 @@ const pointingBeak = ({
 const menuItemStyles: ComponentSlotStylesPrepared<MenuItemPropsAndState, MenuVariables> = {
```

Comment:
Menu's divider and borders are off in dark theme:

![image](https://user-images.githubusercontent.com/4512430/76695880-d3011080-6684-11ea-909a-d1b17583e194.png)

![image](https://user-images.githubusercontent.com/4512430/76695883-ddbba580-6684-11ea-8e51-a7c4ad95d791.png)

Reply:
Never mind it looks like this on master